docs/release_notes/0.0.63.md

```diff
@@ -19,3 +19,5 @@ KM309 🐛 Fix broken access key authentication type for apply cluster (#593)
 KM279 👌 Make integration tests toggleable
 
 KM256 👌 Soften password constrains for cognito users
+
+KM263 👌 Fix grafana https redirect
```

Comment:
Add `(#596)` ? See discussion in https://github.com/oslokommune/okctl/pull/560#discussion_r638795483